### ISSUE TITLE
Backblaze Backup Fails due to updated b2sdk package

### ISSUE THREAD

**Issue by m-picc** [mail-in-a-box/mailinabox#2124]
Issue initially [reported](https://discourse.mailinabox.email/t/has-anyone-got-the-backblaze-b2-backup-working/9106/3) in the mail-in-a-box forum.

The following error occurs when attempting to use backblaze backups with mail-in-a-box:
```
127.0.0.1 - - [19/May/2022 22:30:13] "GET /system/backup/status?_=1653013694457 HTTP/1.0" 200 -
[2022-05-19 22:30:22,333] ERROR in app: Exception on /system/backup/config [POST]
Traceback (most recent call last):
  File "/usr/local/lib/mailinabox/env/lib/python3.6/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/mailinabox/env/lib/python3.6/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/mailinabox/env/lib/python3.6/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/mailinabox/env/lib/python3.6/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/matthewpiccinich/mailinabox/management/daemon.py", line 75, in newview
    return viewfunc(*args, **kwargs)
  File "/home/matthewpiccinich/mailinabox/management/daemon.py", line 637, in backup_set_custom
    request.form.get('min_age', '')
  File "/home/matthewpiccinich/mailinabox/management/backup.py", line 498, in backup_set_custom
    list_target_files(config)
  File "/home/matthewpiccinich/mailinabox/management/backup.py", line 460, in list_target_files
    from b2sdk.v1 import InMemoryAccountInfo, B2Api
  File "/usr/local/lib/mailinabox/env/lib/python3.6/site-packages/b2sdk/v1/__init__.py", line 11, in <module>
    from b2sdk.v2 import *  # noqa
  File "/usr/local/lib/mailinabox/env/lib/python3.6/site-packages/b2sdk/v2/__init__.py", line 11, in <module>
    from b2sdk._v3 import *  # noqa
  File "/usr/local/lib/mailinabox/env/lib/python3.6/site-packages/b2sdk/_v3/__init__.py", line 170, in <module>
    from b2sdk.sync.folder import AbstractFolder
  File "/usr/local/lib/mailinabox/env/lib/python3.6/site-packages/b2sdk/sync/folder.py", line 21, in <module>
    from .scan_policies import DEFAULT_SCAN_MANAGER, ScanPoliciesManager
  File "/usr/local/lib/mailinabox/env/lib/python3.6/site-packages/b2sdk/sync/scan_policies.py", line 108, in <module>
    class ScanPoliciesManager:
  File "/usr/local/lib/mailinabox/env/lib/python3.6/site-packages/b2sdk/sync/scan_policies.py", line 131, in ScanPoliciesManager
    exclude_uploaded_after: Optional[int] = None,
AttributeError: module 're' has no attribute 'Pattern'
 Exception on /system/backup/config [POST]
```

User fracai [explained](https://discourse.mailinabox.email/t/has-anyone-got-the-backblaze-b2-backup-working/9106/5) the following regarding the b2sdk package:

> It looks like re.Pattern was added in [Python 3.7](https://docs.python.org/3.7/library/re.html#re.Pattern.search) and the [b2sdk 1](https://pypi.org/project/b2sdk/) doesn’t support anything lower than that.
> Looks like they dropped support for 3.6 in [April](https://github.com/Backblaze/b2-sdk-python/blob/master/CHANGELOG.md#1150---2022-04-12) so the b2sdk dependency would need to be pinned to 1.14.1.

**SOLUTION**: I’ve been able to get backblaze working by updating the mailinabox/setup/management.sh script at line 33 to specify that it install the b2sdk package at version 1.14.1: `hide_output pip3 install --upgrade b2sdk==1.14.1 boto`
After updating this file, run `sudo mailinabox` again and it will install the b2sdk package in the appropriate location for mailinabox.
After that configure your backblaze settings and perform the backup.

**Comment by m-picc** [comment]
Would it be possible to update the management.sh script with the change at line 33 so users who perform new installs will not experience this issue with setting up backblaze? 

If i wanted to create a pull request with this change, would i have to create a fork of the project first then open a pull request?

Is it not worth fixing since support for ubuntu 22.04 will resolve this issue with an updated version (3.10) of python being used by the distro?

**Comment by JoshData** [comment]
Fixed in #2125.
